src/integration_tests/regression_test.py

```diff
@@ -279,3 +280,24 @@ def solve(a, b):
         r"a - b \mathclose{}\right) - a b"
     )
     _check_function(solve, latex)
+
+
+def test_expand_hypot_function() -> None:
+    def solve(x, y, z):
+        return hypot(x, y, z)
+
+    latex = r"\mathrm{solve}(x, y, z) = " r"\sqrt{x^{{2}} + y^{{2}} + z^{{2}}}"
+    _check_function(solve, latex, expand_functions=["hypot"])
+
+
+def test_expand_nested_function() -> None:
+    def solve(a, b, x, y):
+        return hypot(hypot(a, b), x, y)
+
+    latex = (
+        r"\mathrm{solve}(a, b, x, y) = "
+        r"\sqrt{"
+        r"\sqrt{a^{{2}} + b^{{2}}}^{{2}} + "
+        r"x^{{2}} + y^{{2}}}"
```

Comment:
nit: Maybe it'd be good to apply some symbolic optimizations since nested `hypot` can be removed: $\sqrt{\sqrt{x^2 + y^2}^2 + z^2} = \sqrt{x^2 + y^2 + z^2}$.

This is beyond the scope of this pull request. We may need to make another issue to discuss about it.

Reply:
I would imagine that this would be a separate transformer that takes place before function expansion

Reply:
Yeah, and it would become a more complex optimizer. We could borrow some knowledge from other symbolic math tools as well.